framework/include/utils/InputParameters.h

```diff
@@ -573,6 +573,51 @@ class InputParameters : public Parameters
                        std::vector<std::string> exclude = std::vector<std::string>());
 
   /**
+   * Method for applying common parameters
+   * @param common The set of parameters to apply to the parameters stored in this object
+   * @param include A vector of parameters to apply
+   *
+   * In order to apply common parameter 4 statements must be satisfied
+   *   (1) A local parameter must exist with the same name as common parameter
+   *   (2) Common parameter must valid
+   *   (3) Local parameter must be invalid OR not have been set from its default
+   *   (4) Both cannot be private
+   *
+   * Output objects have a set of common parameters that are passed
+   * down to each of the output objects created. This method is used for
+   * applying those common parameters.
+   *
+   * @see CommonOutputAction AddOutputAction
+   */
+  void applyOnlyParameters(const InputParameters & common,
```

Comment:
I like applyParameter and applyCoupledVar better too.
applySpecificParameters is fine too, and better than applyOnlyParameters.  I don't like the idea of the include/exclude because it makes the user interface more complicated, although i acknowledge it makes the code sleeker.  
"update" instead of "apply" - yea, probably it's more descriptive.  will this have effects on bison and other things i have no control over?